### PR TITLE
build: automate deploy

### PR DESCRIPTION
**Pull request by samsiegart**
fixes https://github.com/Agoric/wallet-app/issues/40

Looking at the whole tree is probably clearer than the diff: https://github.com/Agoric/wallet-app/tree/automate-deploy

See cloudflare deployment working below

### CODE REVIEW

File: package.json
Comment (samsiegart) [inline package.json:14]: This is the crux of the change

File: .github/workflows/test.yml
Comment (turadg) [inline .github/workflows/test.yml:18]: consider `latest`. the build tools will always work on LTS releases.
https://github.com/actions/setup-node#supported-version-syntax

File: .gitignore
Comment (turadg) [inline .gitignore:59]: any reason not to ignore all `complied` and `dist`?

File: README.md
Comment (turadg) [inline README.md:8]: consider `yarn install` for clarity

File: components/install-ses-lockdown.js
Comment (turadg) [inline components/install-ses-lockdown.js:6]: worth a comment on why this is okay here

File: package.json
Comment (turadg) [inline package.json:2]: more specifically,
```suggestion
  "name": "@agoric/wallet-site",
```

File: package.json
Comment (turadg) [inline package.json:82]: this is a private package so let's make maintenance easier on ourselves and use the active LTS.
```suggestion
    "node": ">=18.0"
```

File: wallet/.gitignore
Comment (turadg) [inline wallet/.gitignore:13]: this one got in. please do remove it and massage gitignore so it doesn't get in again

File: wallet/package.json
Comment (turadg) [inline wallet/package.json:2]: We've also been calling this the "wallet app". wallet.agoric.app is where you get it.

"wallet-ui" or "wallet-app" both work for me but whichever you pick please ensure the workspace (parent path) matches the name here.

File: wallet/src/components/SmartWalletConnection.tsx
Comment (turadg) [inline wallet/src/components/SmartWalletConnection.tsx:43]: nice

File: wallet/src/util/keyManagement.ts
Comment (turadg) [inline wallet/src/util/keyManagement.ts:253]: curious what this is. consider some prose explaining the need to suppress.

File: wallet/src/util/querySwingsetParams.ts
Comment (turadg) [inline wallet/src/util/querySwingsetParams.ts:22]: oh is this what the other ones are too? consider filing a ticket in this repo to fix that and linking to it at the sites. second thought, probably not worth the overhead.

**Review by turadg — APPROVED** [review]
Glorious!

Please look over the suggestions and requests before merging. Approving now because I'm so excited to see 58 closed  😸

**Comment by samsiegart** [comment]
I tried addressing some feedback but something broke the cloudflare build and I don't have access to the logs. I'll just merge this so we can tweak from a clean slate.

**Pull request merged**
